### ISSUE TITLE
Unable to install pdpipe in Docker container

### ISSUE THREAD

**Issue by joecorkerton** [shaypal5/pdpipe#2]
With `pip install pdpipe` in the requirements.txt I get the following errors:
```Collecting pdpipe
  Downloading pdpipe-0.0.4.tar.gz
    Complete output from command python setup.py egg_info:
    warning: no files found matching '.isort.cfg'
    warning: no files found matching '.pylintrc'
    warning: no previously-included files matching '*.py[cod]' found anywhere in distribution
    warning: no previously-included files matching '__pycache__' found anywhere in distribution
    warning: no previously-included files matching '*.so' found anywhere in distribution
    
    Installed /tmp/pip-build-9goelx7l/pdpipe/.eggs/pytest_cov-2.5.1-py3.6.egg
    Searching for coverage
    Reading https://pypi.python.org/simple/coverage/
    Downloading https://pypi.python.org/packages/36/db/690ee79312bb60f121c0da1c973856ddb751afe09cc10caec1452208eaf4/coverage-4.4.1.tar.gz#md5=4c4d997cb32be4703f1fde54e6cd1b1c
    Best match: coverage 4.4.1
    Processing coverage-4.4.1.tar.gz
    Writing /tmp/easy_install-ngjlrmjz/coverage-4.4.1/setup.cfg
    Running coverage-4.4.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-ngjlrmjz/coverage-4.4.1/egg-dist-tmp-ixh2ld6d
    warning: no previously-included files found matching 'ci/appveyor.token'
    no previously-included directories found matching 'doc/_build'
    no previously-included directories found matching 'tests/eggsrc/build'
    no previously-included directories found matching 'tests/eggsrc/dist'
    no previously-included directories found matching 'tests/eggsrc/*.egg-info'
    warning: no previously-included files matching '*.py[co]' found anywhere in distribution
    creating /tmp/pip-build-9goelx7l/pdpipe/.eggs/coverage-4.4.1-py3.6-linux-x86_64.egg
    Extracting coverage-4.4.1-py3.6-linux-x86_64.egg to /tmp/pip-build-9goelx7l/pdpipe/.eggs
    
    Installed /tmp/pip-build-9goelx7l/pdpipe/.eggs/coverage-4.4.1-py3.6-linux-x86_64.egg
    Searching for pytest
    Downloading https://pypi.python.org/packages/24/b4/7290d65b2f3633db51393bdf8ae66309b37620bc3ec116c5e357e3e37238/pytest-cov-2.5.1.tar.gz#md5=5acf38d4909e19819eb5c1754fbfc0ac
    Best match: pytest cov-2.5.1
    Processing pytest-cov-2.5.1.tar.gz
    Writing /tmp/easy_install-c0soq7gp/pytest-cov-2.5.1/setup.cfg
    Running pytest-cov-2.5.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-c0soq7gp/pytest-cov-2.5.1/egg-dist-tmp-hyjdeie9
    warning: no files found matching '.isort.cfg'
    warning: no files found matching '.pylintrc'
    warning: no previously-included files matching '*.py[cod]' found anywhere in distribution
    warning: no previously-included files matching '__pycache__' found anywhere in distribution
    warning: no previously-included files matching '*.so' found anywhere in distribution
    removing '/tmp/pip-build-9goelx7l/pdpipe/.eggs/pytest_cov-2.5.1-py3.6.egg' (and everything under it)
    creating /tmp/pip-build-9goelx7l/pdpipe/.eggs/pytest_cov-2.5.1-py3.6.egg
    Extracting pytest_cov-2.5.1-py3.6.egg to /tmp/pip-build-9goelx7l/pdpipe/.eggs
    
    Installed /tmp/pip-build-9goelx7l/pdpipe/.eggs/pytest_cov-2.5.1-py3.6.egg
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-9goelx7l/pdpipe/setup.py", line 52, in <module>
        'Intended Audience :: Developers',
      File "/usr/local/lib/python3.6/distutils/core.py", line 108, in setup
        _setup_distribution = dist = klass(attrs)
      File "/usr/local/lib/python3.6/site-packages/setuptools/dist.py", line 315, in __init__
        self.fetch_build_eggs(attrs['setup_requires'])
      File "/usr/local/lib/python3.6/site-packages/setuptools/dist.py", line 361, in fetch_build_eggs
        replace_conflicting=True,
      File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 853, in resolve
        raise DistributionNotFound(req, requirers)
    pkg_resources.DistributionNotFound: The 'pytest' distribution was not found and is required by the application```

**Comment by shaypal5** [comment]
@joecorkerton Sorry, it was a stupid mistake with `setup_requires` on my part; it copied the test requirements instead of the install ones. I released `0.0.6` with a fix, and I think it should work now. Care to give this a try?

**Comment by joecorkerton** [comment]
Works now!
